policyengine_us/variables/gov/states/wi/tax/income/additions/wi_capital_loss.py

```diff
@@ -15,9 +15,13 @@ class wi_capital_loss(Variable):
 
     def formula(tax_unit, period, parameters):
         # calculate Schedule WD, Line 18
-        GAIN_SOURCES = ["short_term_capital_gains", "long_term_capital_gains"]
-        netcg = add(tax_unit, period, GAIN_SOURCES)
+        netcg = add(
```

Comment:
Why this change? I prefer the original. Or does a variable for this already exist?